sources/packages/backend/workflow/test/2022-2023/parttime-assessment/disbursements/parttime-assessment-disbursements.e2e-spec.ts

```diff
@@ -473,7 +473,7 @@ describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-disbursements`,
               },
               {
                 awardEligibility: true,
-                valueAmount: testFinalAwardNetAmount.secondDisbursementAmount,
+                valueAmount: 0,
```

Comment:
Is there a reason not to have this value adjusted in the `testFinalAwardNetAmount`?